x-pack/plugins/security_solution/public/cases/containers/api.test.tsx

```diff
@@ -395,6 +395,7 @@ describe('Case Configuration API', () => {
       settings: {
         syncAlerts: true,
       },
+      // TODO: remove this
```

Comment:
Why?

Reply:
Thanks, forgot that this was a test 😆 , I'll remove it.